initfiles/bin/init_thorslave.in

```diff
@@ -65,7 +65,11 @@ start_slaves()
 {
     # insuring dafilesrv is running on the machine as it is a prerequisite
     dafilesrv_owner=$(stat -c "%U" ${INIT_PATH}/dafilesrv 2>/dev/null || stat -f %Su ${INIT_PATH}/dafilesrv 2>/dev/null)
-    [[ "${dafilesrv_owner}" != "${user}" ]] && cmd_prefix="sudo"
+    # what is stat -f %Su and why check file owner ?
```

Comment:
It's part of an osx compatibility patch.  See https://github.com/hpcc-systems/HPCC-Platform/commit/007b784d062a859181e2aacf50237f7111251649